cmd/sprint-automation/main.go

```diff
@@ -562,20 +562,28 @@ func sendTriageBuild02Upgrade(slackClient *slack.Client, version, stableDuration
 	return nil
 }
 
+const dateFormat = "Mon, 02 Jan 2006"
+
 func blockForIssue(issue jiraapi.Issue) slack.Block {
 	// we really don't want these things to line wrap, so truncate the summary
 	cutoff := 85
 	summary := issue.Fields.Summary
 	if len(summary) > cutoff {
 		summary = summary[0:cutoff-3] + "..."
 	}
+	created := time.Time(issue.Fields.Created).Format(dateFormat)
+	updated := time.Time(issue.Fields.Updated).Format(dateFormat)
 	return &slack.ContextBlock{
 		Type: slack.MBTContext,
 		ContextElements: slack.ContextElements{
 			Elements: []slack.MixedElement{
 				&slack.TextBlockObject{
 					Type: slack.MarkdownType,
-					Text: fmt.Sprintf("<https://issues.redhat.com/browse/%s|*%s*>: %s", issue.Key, issue.Key, summary),
+					Text: fmt.Sprintf("<https://issues.redhat.com/browse/%s|*%s*>: %s \n", issue.Key, issue.Key, summary),
+				},
+				&slack.TextBlockObject{
+					Type: slack.PlainTextType,
+					Text: fmt.Sprintf("Created on: %s  Last updated: %s", created, updated),
```

Comment:
Is it easy to get the author who did the update?
Do we want to avoid robot updates?

Reply:
We do have access to the `changelog` so we could theoretically ignore updates by certain users in this digest. I do wonder where robot updates fit into our workflow though...would we want to not surface those as an updated card? For intake, would we not want to see that there had been a change just because it was a robot change?

I could also potentially add the date of the most recent comment here (assuming one exists), but the overall goal was just to surface things that we **know** haven't changed since we have last looked.

Reply:
/hold cancel
I will look into adding the `updateBy`, but I will do it in a follow up PR. I am intake this week, and I want to see this in action tomorrow :)